emcc.py

```diff
@@ -2825,9 +2825,15 @@ def check_bad_eq(arg):
         if requested_level.startswith('force_dwarf'):
           exit_with_error('gforce_dwarf was a temporary option and is no longer necessary (use -g)')
         elif requested_level.startswith('separate-dwarf'):
-          # Emit full DWARF but also emit it in a file on the side
+          # emit full DWARF but also emit it in a file on the side
           newargs[i] = '-g'
-          shared.Settings.SEPARATE_DWARF = 1
+          # if a file is provided, use that; otherwise use the default location
+          # (note that we do not know the default location until all args have
+          # been parsed, so just note True for now).
+          if '=' in requested_level:
+            shared.Settings.SEPARATE_DWARF = requested_level.split('=')[1]
```

Comment:
should we assert that there is only 1 element to the right of the first split? (or equivalently that there are no '=' in the filename?

Reply:
Can't hurt, sure, adding.